openskistats/tests/test_bearing.py

```diff
@@ -69,92 +69,92 @@ class BearingSummaryStatsPytestParam:
     [
         BearingSummaryStatsPytestParam(
             bearings=[0.0],
-            weights=[2.0],
             combined_vertical=[2.0],
+            alignments=[1.0],
             hemisphere="north",
             expected_bearing=0.0,
-            expected_strength=1.0,
+            expected_alignment=1.0,
             expected_poleward_affinity=1.0,
             excepted_eastward_affinity=0.0,
         ),
         BearingSummaryStatsPytestParam(
             bearings=[0.0, 90.0],
-            weights=[1.0, 1.0],
             combined_vertical=None,
+            alignments=[1.0, 1.0],
             hemisphere="south",
             expected_bearing=45.0,
-            expected_strength=0.7071068,
+            expected_alignment=0.7071068,
             expected_poleward_affinity=-0.5,
             excepted_eastward_affinity=0.5,
         ),
         BearingSummaryStatsPytestParam(
             bearings=[0.0, 90.0],
-            weights=[0.5, 0.5],
             combined_vertical=[0.5, 0.5],
+            alignments=[1.0, 1.0],
             hemisphere="north",
             expected_bearing=45.0,
-            expected_strength=0.7071068,
+            expected_alignment=0.7071068,
             expected_poleward_affinity=0.5,
             excepted_eastward_affinity=0.5,
         ),
         BearingSummaryStatsPytestParam(
             bearings=[0.0, 90.0],
-            weights=None,
             combined_vertical=[2.0, 2.0],
+            alignments=None,
             hemisphere="north",
             expected_bearing=45.0,
-            expected_strength=0.3535534,
-            expected_poleward_affinity=0.25,
-            excepted_eastward_affinity=0.25,
+            expected_alignment=0.7071068,
+            expected_poleward_affinity=0.5,
+            excepted_eastward_affinity=0.5,
         ),
         BearingSummaryStatsPytestParam(
             bearings=[0.0, 360.0],
-            weights=[1.0, 1.0],
             combined_vertical=None,
+            alignments=[1.0, 1.0],
             hemisphere="north",
             expected_bearing=0.0,
-            expected_strength=1.0,
+            expected_alignment=1.0,
             expected_poleward_affinity=1.0,
             excepted_eastward_affinity=0.0,
         ),
         BearingSummaryStatsPytestParam(
             bearings=[0.0, 90.0],
-            weights=[0.0, 1.0],
             combined_vertical=[0.5, 1.5],
+            alignments=[0.0, 1.0],
             hemisphere="north",
             expected_bearing=90.0,
-            expected_strength=0.5,
+            expected_alignment=0.75,
             expected_poleward_affinity=0.0,
-            excepted_eastward_affinity=0.5,
+            excepted_eastward_affinity=0.75,
```

Comment:
Similar here. For the second segment, a 1.0 alignment is the same as old weight = 1.5, which is 1.5x its value before. So the results are scaled by 1.5.